From 316242281eeef1e32640b07a533c91c90c5397a1 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 6 Feb 2019 12:48:51 +0000
Subject: [PATCH 0868/1000] MANTIS 0024715 Build: VC10; fix compiler warnings

warning C4091: 'extern __declspec(dllimport)' : ignored on left of 'double' when no variable is declared

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1433 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/cpp/XmlWriter/src/XmlWriter.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/SelfServiceCommon/Massai/cpp/XmlWriter/src/XmlWriter.cpp b/SelfServiceCommon/Massai/cpp/XmlWriter/src/XmlWriter.cpp
index 59228305..164cef47 100644
--- a/SelfServiceCommon/Massai/cpp/XmlWriter/src/XmlWriter.cpp
+++ b/SelfServiceCommon/Massai/cpp/XmlWriter/src/XmlWriter.cpp
@@ -2,6 +2,7 @@
 //
 //////////////////////////////////////////////////////////////////////
 
+#include <math.h>
 #include "XmlWriter/XmlWriter.h"
 #include <crtdbg.h>
 
-- 
2.41.0.windows.1

